project/BuildSettings.scala

```diff
@@ -25,13 +31,13 @@ object BuildSettings {
     description := "Functional programming toolkit for building server applications in Scala.",
     licenses := Seq("MIT" -> url("https://raw.githubusercontent.com/avast/scala-server-toolkit/master/LICENSE")),
     developers := List(Developer("jakubjanecek", "Jakub Janecek", "janecek@avast.com", url("https://www.avast.com"))),
-    scalaVersion := "2.13.8",
-    crossScalaVersions := List(scalaVersion.value, "2.12.15"),
+    scalaVersion := scala3,
```

Comment:
I'm not sure about having Scala 3 as the default is a good idea.
Wouldn't having 2.13 as the default save us a lot of headache with Scalafix (among other things)?

Reply:
idk, in my view scala3 is the future so i would vote for shaping the code towards that ... i would also like to move further towards scala3 code in future PRs using compiler toggles etc

Reply:
My first choice would probably also be `2.13` for now but if everything works and we can compile to both versions the default is probably not that important.

Reply:
it only makes a big difference for mdoc imho ... if its all the same to you, i would try to coarse the code into scala3 compatible code as much as possible so i think scala3 default is the best way to go